.github/workflows/go.yml

```
@@ -30,17 +30,20 @@ jobs:
           key: ${{ runner.os }}-go-${{ matrix.go }}-${{ hashFiles('tests/go.mod') }}
 
       - name: Build
-        run: CGO_ENABLED=0 find . -name go.mod -execdir go build ./... \;
+        run: find . -name go.mod -execdir sh -c 'pwd && CGO_ENABLED=0 go build ./... || exit 1' sh {} +
+
+      - name: JS/WASM Build
+        run: find . -name go.mod -not -path "*examples*" -execdir sh -c 'pwd && GOOS=js GOARCH=wasm CGO_CENABLED=0 go build ./... || exit 1' sh {} +
```

This seems wrong. I don't think you should need to set CGO_ENABLED=0 for wasm anyway.
```suggestion
        run: find . -name go.mod -not -path "*examples*" -execdir sh -c 'pwd && GOOS=js GOARCH=wasm CGO_ENABLED=0 go build ./... || exit 1' sh {} +
```

---

Ha, yep, that was my bad. And it's not necessary I don't think.